### DISCUSSION TITLE
ghcr vs docker hub

### DISCUSSION THREAD

**Discussion by ShahriarKh** [amacneil/dbmate#593] [Support] [answered]
Per docs:
> _Docker images are published to GitHub Container Registry ([ghcr.io/amacneil/dbmate](https://ghcr.io/amacneil/dbmate))._

But there's https://hub.docker.com/r/amacneil/dbmate too.

Is there a difference? If not, I guess the docs are kinda misleading.

**Comment by amacneil** [comment]
We currently push to both, but GHCR is recommended.

**Reply by ShahriarKh** [reply]
I see, I see... but, any specific reason? Just wondering.

**Reply by amacneil** [reply, marked as answer]
I think at some point docker hub started charging people to pull images, so I started publishing to ghcr as an alternative.

I was planning to deprecate docker hub but it seemed like that should happen on a major version bump (so as not to disrupt ppl expecting the latest image there), and we haven't had a major version bump since then. 

So for now docker hub just lives in this up-to-date-but-undocumented state 🤷‍♀️
